bin/ch/ch.cpp

```diff
@@ -861,6 +861,7 @@ int _cdecl RunJITServer(int argc, __in_ecount(argc) LPWSTR argv[])
     DWORD status = UuidFromStringW((RPC_WSTR)connectionUuidString, &connectionUuid);
     if (status != RPC_S_OK)
     {
+        ChakraRTInterface::UnloadChakraDll(chakraLibrary);
```

Comment:
I'd rather not duplicate this call like this, especially since we're duplicating it without duplicating the check for its non-null-ness (as seen on L877). Could you instead make this function single-return and have a `goto cleanup` here instead? That way we'd avoid having different or duplicated logic about when to clean-up chakraLibrary.